gcm/README.md

```diff
@@ -12,7 +12,6 @@ Add dependency to the application level `build.gradle` file.
 
 ```groovy
 dependencies {
-    implementation 'com.parse:parse-android:latest.version.here'
     implementation 'com.parse:parse-android-gcm:latest.version.here'
```

Comment:
https://api.bintray.com/packages/parse/maven/com.parse:parse-android/images/download.svg should be added as well